shared/src/models/utils/cater-for.ts

```diff
@@ -1,17 +1,35 @@
-import { IsNumber, IsString } from 'class-validator';
+import { IsNumber, IsString, Min } from 'class-validator';
 import { getCreate } from './get-create';
 
 export class CanCaterFor {
+    /**
+     * if {@link logicalOperator}  `=== 'and'` it is cumulative,
+     * how many red patients can the catering treat
+     * also affects the number of possible {@link yellow} and {@link green} patients
+     * that can be treated.
+     */
     @IsNumber()
+    @Min(0)
     public readonly red: number;
 
+    /**
+     * if {@link logicalOperator}  `=== 'and'` it is cumulative,
+     * how many extra {@link yellow} and {@link green} patients can the catering treat
+     * to the number already written in the {@link red} value
+     */
     @IsNumber()
+    @Min(0)
     public readonly yellow: number;
 
+    /**
+     * if {@link logicalOperator}  `=== 'and'` it is cumulative,
+     * how many {@link green} patients can the catering treat
+     * to the number already written in the {@link yellow} and {@link red} value
+     */
     @IsNumber()
+    @Min(0)
     public readonly green: number;
 
-    // TODO
```

Comment:
Why is this comment removed? I think I've marked this spot in a different way in #125 but I'd like to keep this comment here.

Reply:
A TODO comment should always state what should be done. I could imagine I removed it because during some refactoring in this bloated PR because I could not identify what it meant. With the context of the stricter validation it makes sense.

I'll add it back again.

Reply:
Fixed with 4e542b4f842f7fd1c7afa51a2d20846181845808